src/test/java/org/sagebionetworks/bridge/validators/SubpopulationValidatorTest.java

```diff
@@ -24,11 +24,10 @@
     
     @BeforeMethod
     public void before() {
-        validator = new SubpopulationValidator(TestConstants.USER_DATA_GROUPS, TestConstants.USER_STUDY_IDS);
+        validator = new SubpopulationValidator(USER_DATA_GROUPS, USER_STUDY_IDS);
     }
     
-    @Test
-    public void testEntirelyValid() {
+    Subpopulation getSubpopulation() { 
```

Comment:
Removed. I had split this test up into testing each field separately, which would be better, but then decided against it because it just added more changes to review.